diff --git a/dep/gsoap/stdsoap2.cpp b/dep/gsoap/stdsoap2.cpp
index ed0922d..3d51af4 100644
--- a/dep/gsoap/stdsoap2.cpp
+++ b/dep/gsoap/stdsoap2.cpp
@@ -15245,10 +15245,12 @@ soap_puthttphdr(struct soap *soap, int status, size_t count)
       }
       s = soap->tmpbuf;
     }
+/*
     if (status == SOAP_OK && soap->version == 2 && soap->action && strlen(soap->action) < sizeof(soap->tmpbuf) - 80)
     { sprintf(soap->tmpbuf, "%s; action=\"%s\"", s, soap->action);
       s = soap->tmpbuf;
     }
+*/
 #endif
     if (s && (err = soap->fposthdr(soap, "Content-Type", s)))
       return err;
